src/Microsoft.TestPlatform.Extensions.BlameDataCollector/Win32NamedEvent.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.Extensions.BlameDataCollector
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    public class Win32NamedEvent
```

Comment:
Should this just be part of the NativeMethodHelper?

Reply:
i would not want to dump all native methods in the same class for no reason